File: TP1 - C++/data/models/payment.cpp
#include "payment.hpp"

Payment::Payment(string id, string date, double value, string description):
    id(id),
    date(date),
    value(value),
    description(description){};

Payment::~Payment(){};

string Payment::getId(){
    return this->id;
}

string Payment::getDate(){
    return this->date;
}

double Payment::getValue(){
    return this->value;
}

string Payment::getDescription(){
    return this->description;
}

void Payment::setDate(string date){
    this->date = date;
}

void Payment::setValue(double value){
    this->value = value;
}

void Payment::setDescription(string description){
    this->description = description;
}

File: TP2 - Java/src/domain/entities/Payment.java
package domain.entities;

public class Payment {
  private final String id;
  private String date;
  private double value;
  private String description;

  public Payment(String id, String date, double value, String description) {
    this.id = id;
    this.date = date;
    this.value = value;
    this.description = description;
  }

  public String getId() {
    return id;
  }

  public String getDate() {
    return date;
  }

  public void setDate(String date) {
    this.date = date;
  }

  public double getValue() {
    return value;
  }

  public void setValue(double value) {
    this.value = value;
  }

  public String getDescription() {
    return description;
  }

  public void setDescription(String description) {
    this.description = description;
  }
}
